python/lsst/pipe/tasks/makeCoaddTempExp.py

```diff
@@ -538,3 +553,194 @@ def applySkyCorr(self, dataRef, calexp):
         if isinstance(calexp, afwImage.Exposure):
             calexp = calexp.getMaskedImage()
         calexp -= bg.getImage()
+
+
+class MakeWarpConfig(pipeBase.PipelineTaskConfig, MakeCoaddTempExpConfig):
+    calExpList = pipeBase.InputDatasetField(
+        doc="Exposures which are resampled and optionally PSF-Matched onto SkyMap projection and pixelation",
+        name="calexp",
+        storageClass="ExposureF",
+        dimensions=("Visit", "Detector")
+    )
+    backgroundList = pipeBase.InputDatasetField(
+        doc="Backgrounds to optionally add back into the calexp",
+        name="calexpBackground",
+        storageClass="Background",
+        dimensions=("Visit", "Detector")
+    )
+    skyCorrList = pipeBase.InputDatasetField(
+        doc="SkyCorr",
```

Comment:
We should remember to go back and expand these docstrings later if we're not doing them now.  If you don't have a pre-existing list for such things (ticket or otherwise) I can start one.

Reply:
I had entirely intended to go back and write complete docstrings for these on this ticket.